doc/driver.mld

```diff
@@ -132,20 +132,20 @@ In this section [odoc] is used to generate the documentation of [odoc] and some
 Let's start with some functions to execute the three phases of [odoc].
 
 Compiling a file with [odoc] requires a few arguments: the file to compile, an
-optional parent, a list of include paths, a list of children for [.mld] files
+optional parent, a list of include paths, a list of children for [.mld] files,
 and an output path. Include paths can be just ['.'], and we can calculate the
-output file from the input because all of the files are going into the same
+output file from the input because all the files go into the same
 directory.
 
-Odoc supports compiling {i implementations}. This can be useful to render source
+[odoc] supports compiling {i implementations}. This can be useful to render source
 code, generate links from and to source code, and count occurrences of
-identifiers. The [compile-src] command takes a source-tree parent file, and a
-source path which has to be in the source tree.
+identifiers. The [compile-src] command takes a source-tree parent file and a
+source path, which has to be in the source tree.
```

Comment:
I preferred the previous version: it made more clear that the "has to be in the source tree" refers to the "source path".

Reply:
So would it be better to say:
"The [compile-src] command takes a source-tree parent file and a
source path that has to be in the source tree."

Reply:
Yes!